stl/inc/chrono

```diff
@@ -2836,6 +2875,13 @@ namespace chrono {
         }
     }
 
+    // FUNCTION remote_version
+    _NODISCARD inline string remote_version() {
+        const auto& _Version            = get_tzdb().version;
+        const auto [_Leap_sec, _Ignore] = _Xtzdb_generate_leap_seconds(get_tzdb().leap_seconds.size());

```

Comment:
This calls `get_tzdb()` twice, which I think is inefficient - it should be extracted.